python/GafferUI/GraphBookmarksUI.py

```diff
@@ -130,11 +130,19 @@ def followBookmark( number, weakEditor, _ ) :
 
 	n = editor.getNodeSet()
 
+	script = editor.ancestor( GafferUI.ScriptWindow ).scriptNode()
+
+	menuDefinition.append( "/NumericBookmarkDivider", { "divider" : True, "label" : "Numeric Bookmark" } )
```

Comment:
Perhaps "_Follow_ Numeric Bookmark"? Because the section above is "Follow", opening a new section would seem to imply that the subsequent items aren't follow items.

Reply:
Updated [here](https://github.com/GafferHQ/gaffer/pull/4085/commits/1d37a4d2842ccdc026cad810d233bda78ee8daa3).